llpc/test/shaderdb/relocatable_shaders/TriangleVs_CheckNoteSectionForCacheHash.spvasm

```diff
@@ -1,3 +1,18 @@
+; This test case checks that the 128-bit cache hash is correctly added to the PAL metadata.
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -gfxip=9 -unlinked -enable-relocatable-shader-elf -v -o %t.elf %s | FileCheck --check-prefix=CHECK1 %s
+
+; CHECK1-LABEL: {{^// LLPC}} calculated hash results (graphics pipeline)
+; CHECK1-LABEL: Building pipeline with relocatable shader elf.
+; CHECK1-NEXT:  LLPC version: [[llpc_version:([0-9a-f]{2} ?){8}]]{{$}}
+; CHECK1:       Hash for vertex stage cache lookup: [[llpc_vert_hash:([a-f0-9]{2} ?){16}]]{{$}}
+; CHECK1-LABEL: {{^// LLPC}} SPIRV-to-LLVM translation results
+
+; CHECK1-LABEL: {{^=====}}  AMDLLPC SUCCESS  =====
+
+; CHECK1: 128_bit_cache_hash: [ [[llpc_vert_hash]]{{$}} ]
```

Comment:
@kuhar This test is not working yet.  I'm not sure of a good way to do this match.  I know you changed the original code to use some magic to make it smaller.  Is there more magic we can use for the new test.

The output we want to make will be something like

```
        .xgl_cache_info: {
            .128_bit_cache_hash: [ 0x00000000000000BF 0x000000000000009E 0x000000000000002B 0x000000000000006D 0x000000000000006E 0x000000000000007F 0x000000000000008C 0x0000000000000066 0x000000000000009F 0x00000000000000F7 0x000000000000008E 0x0000000000000006 0x000000000000000C 0x00000000000000E8 0x00000000000000AE 0x000000000000004B ]
            .llpc_version: [ 0x000000000000002E 0x0000000000000001 ]}}]
```

Reply:
If the two print formats don't match, I think you would have to match each byte group separately. If the output format of these byte groups are different, you can do a numeric match with https://llvm.org/docs/CommandGuide/FileCheck.html#filecheck-numeric-substitution-blocks.

BTW, this check line has a wrong format. `[[var_name]]` means that you are expecting to see some existing named match, while `{{$}}` means that you want to match the end line sequence without storing it to any variable. See the FileCheck reference from the link above for more details.

Reply:
Thanks.  I had to match them separately.  I used the numeric match.  That worked well thanks.